app/controllers/users_controller.rb

```diff
@@ -31,22 +32,34 @@ def edit
   # POST /users.json
   def create
     @user = User.new(user_params)
-    user = @user
     @user.profile_picture.attach(user_params[:profile_picture])
     if @user.username === ""
       @user.username = @user.name + rand((User.all.length)..(User.all.length * 100000000)).to_s
       @user.save
     end
-    respond_to do |format|
-      if @user.save
+    if @user.birthdate.is_a? String
+        @user.birthdate = Date.parse(@user.birthdate)
```

Comment:
We shouldn't pass the entire birthday as a string. This is fragile and prone to breaking. We should have different attributes for birthday month, day, and year. Month should preferably be a number as well.

Reply:
So, you say we should pass the month, day and year, as separated columns in the database as well, or just in the frontend?

Reply:
We can store the date in one column in the backend, but the frontend should give us the date separate.